{
  "uid" : "3a7e7d0df25be538",
  "name" : "Verify content of the text in sections 1, 2",
  "fullName" : "tests.start_unauthorized_page_test.TestStartUnauthorizedPageText#test_su_02_03_verify_page_text",
  "historyId" : "b87373ac223bbd10e938004c8d671bff",
  "time" : {
    "start" : 1728781339737,
    "stop" : 1728781339737,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55bbe16c402a <unknown>\n#1 0x55bbe13aa5e0 <unknown>\n#2 0x55bbe13a27eb <unknown>\n#3 0x55bbe1392a09 <unknown>\n#4 0x55bbe1394744 <unknown>\n#5 0x55bbe1392cbd <unknown>\n#6 0x55bbe139252f <unknown>\n#7 0x55bbe1392422 <unknown>\n#8 0x55bbe1390493 <unknown>\n#9 0x55bbe1390ada <unknown>\n#10 0x55bbe13acf41 <unknown>\n#11 0x55bbe143e8c5 <unknown>\n#12 0x55bbe141eb22 <unknown>\n#13 0x55bbe143dd7d <unknown>\n#14 0x55bbe141e8c3 <unknown>\n#15 0x55bbe13ec6b3 <unknown>\n#16 0x55bbe13ed68e <unknown>\n#17 0x55bbe168ea2b <unknown>\n#18 0x55bbe16929b1 <unknown>\n#19 0x55bbe167b225 <unknown>\n#20 0x55bbe1693532 <unknown>\n#21 0x55bbe166038f <unknown>\n#22 0x55bbe16b2f28 <unknown>\n#23 0x55bbe16b30f3 <unknown>\n#24 0x55bbe16c2e7c <unknown>\n#25 0x7fd99c694ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"7de2b72bec7e25e195543405b0bd574d\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c0fc86b0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55bbe16b30f3 \\\\u003Cunknown>\\\\n#24 0x55bbe16c2e7c \\\\u003Cunknown>\\\\n#25 0x7fd99c694ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55bbe16c402a <unknown>\nE       #1 0x55bbe13aa5e0 <unknown>\nE       #2 0x55bbe13a27eb <unknown>\nE       #3 0x55bbe1392a09 <unknown>\nE       #4 0x55bbe1394744 <unknown>\nE       #5 0x55bbe1392cbd <unknown>\nE       #6 0x55bbe139252f <unknown>\nE       #7 0x55bbe1392422 <unknown>\nE       #8 0x55bbe1390493 <unknown>\nE       #9 0x55bbe1390ada <unknown>\nE       #10 0x55bbe13acf41 <unknown>\nE       #11 0x55bbe143e8c5 <unknown>\nE       #12 0x55bbe141eb22 <unknown>\nE       #13 0x55bbe143dd7d <unknown>\nE       #14 0x55bbe141e8c3 <unknown>\nE       #15 0x55bbe13ec6b3 <unknown>\nE       #16 0x55bbe13ed68e <unknown>\nE       #17 0x55bbe168ea2b <unknown>\nE       #18 0x55bbe16929b1 <unknown>\nE       #19 0x55bbe167b225 <unknown>\nE       #20 0x55bbe1693532 <unknown>\nE       #21 0x55bbe166038f <unknown>\nE       #22 0x55bbe16b2f28 <unknown>\nE       #23 0x55bbe16b30f3 <unknown>\nE       #24 0x55bbe16c2e7c <unknown>\nE       #25 0x7fd99c694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781340519,
      "stop" : 1728781341111,
      "duration" : 592
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55bbe16c402a <unknown>\n#1 0x55bbe13aa5e0 <unknown>\n#2 0x55bbe13a27eb <unknown>\n#3 0x55bbe1392a09 <unknown>\n#4 0x55bbe1394744 <unknown>\n#5 0x55bbe1392cbd <unknown>\n#6 0x55bbe139252f <unknown>\n#7 0x55bbe1392422 <unknown>\n#8 0x55bbe1390493 <unknown>\n#9 0x55bbe1390ada <unknown>\n#10 0x55bbe13acf41 <unknown>\n#11 0x55bbe143e8c5 <unknown>\n#12 0x55bbe141eb22 <unknown>\n#13 0x55bbe143dd7d <unknown>\n#14 0x55bbe141e8c3 <unknown>\n#15 0x55bbe13ec6b3 <unknown>\n#16 0x55bbe13ed68e <unknown>\n#17 0x55bbe168ea2b <unknown>\n#18 0x55bbe16929b1 <unknown>\n#19 0x55bbe167b225 <unknown>\n#20 0x55bbe1693532 <unknown>\n#21 0x55bbe166038f <unknown>\n#22 0x55bbe16b2f28 <unknown>\n#23 0x55bbe16b30f3 <unknown>\n#24 0x55bbe16c2e7c <unknown>\n#25 0x7fd99c694ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781340519,
        "stop" : 1728781341111,
        "duration" : 592
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55bbe16c402a <unknown>\n#1 0x55bbe13aa5e0 <unknown>\n#2 0x55bbe13a27eb <unknown>\n#3 0x55bbe1392a09 <unknown>\n#4 0x55bbe1394744 <unknown>\n#5 0x55bbe1392cbd <unknown>\n#6 0x55bbe139252f <unknown>\n#7 0x55bbe1392422 <unknown>\n#8 0x55bbe1390493 <unknown>\n#9 0x55bbe1390ada <unknown>\n#10 0x55bbe13acf41 <unknown>\n#11 0x55bbe143e8c5 <unknown>\n#12 0x55bbe141eb22 <unknown>\n#13 0x55bbe143dd7d <unknown>\n#14 0x55bbe141e8c3 <unknown>\n#15 0x55bbe13ec6b3 <unknown>\n#16 0x55bbe13ed68e <unknown>\n#17 0x55bbe168ea2b <unknown>\n#18 0x55bbe16929b1 <unknown>\n#19 0x55bbe167b225 <unknown>\n#20 0x55bbe1693532 <unknown>\n#21 0x55bbe166038f <unknown>\n#22 0x55bbe16b2f28 <unknown>\n#23 0x55bbe16b30f3 <unknown>\n#24 0x55bbe16c2e7c <unknown>\n#25 0x7fd99c694ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"7de2b72bec7e25e195543405b0bd574d\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781339737,
      "stop" : 1728781340519,
      "duration" : 782
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781341144,
      "stop" : 1728781341206,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Start Unauthorized Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "start_unauthorized_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestStartUnauthorizedPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.start_unauthorized_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 6,
        "skipped" : 0,
        "passed" : 400,
        "unknown" : 0,
        "total" : 406
      },
      "items" : [ {
        "uid" : "b916e626df2552f5",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/b916e626df2552f5",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55a45ea6c02a <unknown>\n#1 0x55a45e7525e0 <unknown>\n#2 0x55a45e74a7eb <unknown>\n#3 0x55a45e73aa09 <unknown>\n#4 0x55a45e73c744 <unknown>\n#5 0x55a45e73acbd <unknown>\n#6 0x55a45e73a52f <unknown>\n#7 0x55a45e73a422 <unknown>\n#8 0x55a45e738493 <unknown>\n#9 0x55a45e738ada <unknown>\n#10 0x55a45e754f41 <unknown>\n#11 0x55a45e7e68c5 <unknown>\n#12 0x55a45e7c6b22 <unknown>\n#13 0x55a45e7e5d7d <unknown>\n#14 0x55a45e7c68c3 <unknown>\n#15 0x55a45e7946b3 <unknown>\n#16 0x55a45e79568e <unknown>\n#17 0x55a45ea36a2b <unknown>\n#18 0x55a45ea3a9b1 <unknown>\n#19 0x55a45ea23225 <unknown>\n#20 0x55a45ea3b532 <unknown>\n#21 0x55a45ea0838f <unknown>\n#22 0x55a45ea5af28 <unknown>\n#23 0x55a45ea5b0f3 <unknown>\n#24 0x55a45ea6ae7c <unknown>\n#25 0x7f6d9fc94ac3 <unknown>",
        "time" : {
          "start" : 1728694665763,
          "stop" : 1728694665763,
          "duration" : 0
        }
      }, {
        "uid" : "6e597b9428da1920",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/6e597b9428da1920",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56016c5f402a <unknown>\n#1 0x56016c2da5e0 <unknown>\n#2 0x56016c2d27eb <unknown>\n#3 0x56016c2c2a09 <unknown>\n#4 0x56016c2c4744 <unknown>\n#5 0x56016c2c2cbd <unknown>\n#6 0x56016c2c252f <unknown>\n#7 0x56016c2c2422 <unknown>\n#8 0x56016c2c0493 <unknown>\n#9 0x56016c2c0ada <unknown>\n#10 0x56016c2dcf41 <unknown>\n#11 0x56016c36e8c5 <unknown>\n#12 0x56016c34eb22 <unknown>\n#13 0x56016c36dd7d <unknown>\n#14 0x56016c34e8c3 <unknown>\n#15 0x56016c31c6b3 <unknown>\n#16 0x56016c31d68e <unknown>\n#17 0x56016c5bea2b <unknown>\n#18 0x56016c5c29b1 <unknown>\n#19 0x56016c5ab225 <unknown>\n#20 0x56016c5c3532 <unknown>\n#21 0x56016c59038f <unknown>\n#22 0x56016c5e2f28 <unknown>\n#23 0x56016c5e30f3 <unknown>\n#24 0x56016c5f2e7c <unknown>\n#25 0x7f3fe8894ac3 <unknown>",
        "time" : {
          "start" : 1728608268692,
          "stop" : 1728608268692,
          "duration" : 0
        }
      }, {
        "uid" : "ad72082b8c07207f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/ad72082b8c07207f",
        "status" : "passed",
        "time" : {
          "start" : 1728557426223,
          "stop" : 1728557426293,
          "duration" : 70
        }
      }, {
        "uid" : "bc1c5df9d29318ec",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/bc1c5df9d29318ec",
        "status" : "passed",
        "time" : {
          "start" : 1728554292148,
          "stop" : 1728554292229,
          "duration" : 81
        }
      }, {
        "uid" : "323c42838b07ebb7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/323c42838b07ebb7",
        "status" : "passed",
        "time" : {
          "start" : 1728522301329,
          "stop" : 1728522301401,
          "duration" : 72
        }
      }, {
        "uid" : "108799c21fdf1269",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/108799c21fdf1269",
        "status" : "passed",
        "time" : {
          "start" : 1728470686631,
          "stop" : 1728470686711,
          "duration" : 80
        }
      }, {
        "uid" : "902c926c63a77afc",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/902c926c63a77afc",
        "status" : "passed",
        "time" : {
          "start" : 1728468116318,
          "stop" : 1728468116424,
          "duration" : 106
        }
      }, {
        "uid" : "864540327c0ebded",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/864540327c0ebded",
        "status" : "passed",
        "time" : {
          "start" : 1728435928434,
          "stop" : 1728435928517,
          "duration" : 83
        }
      }, {
        "uid" : "f4ada3880e068ebf",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/f4ada3880e068ebf",
        "status" : "passed",
        "time" : {
          "start" : 1728378129878,
          "stop" : 1728378129951,
          "duration" : 73
        }
      }, {
        "uid" : "5d941e64dd62373c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/5d941e64dd62373c",
        "status" : "passed",
        "time" : {
          "start" : 1728375339447,
          "stop" : 1728375339521,
          "duration" : 74
        }
      }, {
        "uid" : "815ad999d7d92d77",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/815ad999d7d92d77",
        "status" : "passed",
        "time" : {
          "start" : 1728375220804,
          "stop" : 1728375220892,
          "duration" : 88
        }
      }, {
        "uid" : "9de2d37bd828ac9f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/9de2d37bd828ac9f",
        "status" : "passed",
        "time" : {
          "start" : 1728349309045,
          "stop" : 1728349309128,
          "duration" : 83
        }
      }, {
        "uid" : "db1128937149f375",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/db1128937149f375",
        "status" : "passed",
        "time" : {
          "start" : 1728293058764,
          "stop" : 1728293058845,
          "duration" : 81
        }
      }, {
        "uid" : "bc067c96cbcacfbd",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/bc067c96cbcacfbd",
        "status" : "passed",
        "time" : {
          "start" : 1728290109492,
          "stop" : 1728290109563,
          "duration" : 71
        }
      }, {
        "uid" : "7a660a85e606216b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/7a660a85e606216b",
        "status" : "passed",
        "time" : {
          "start" : 1728263756057,
          "stop" : 1728263756126,
          "duration" : 69
        }
      }, {
        "uid" : "c16134bc4a595ad9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/c16134bc4a595ad9",
        "status" : "passed",
        "time" : {
          "start" : 1728226937460,
          "stop" : 1728226937540,
          "duration" : 80
        }
      }, {
        "uid" : "f90289c5b82ef707",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/f90289c5b82ef707",
        "status" : "passed",
        "time" : {
          "start" : 1728222024977,
          "stop" : 1728222025071,
          "duration" : 94
        }
      }, {
        "uid" : "6f08e8243fb48b27",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/6f08e8243fb48b27",
        "status" : "passed",
        "time" : {
          "start" : 1728177121166,
          "stop" : 1728177121244,
          "duration" : 78
        }
      }, {
        "uid" : "a2845806b4c39122",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/a2845806b4c39122",
        "status" : "passed",
        "time" : {
          "start" : 1728117752993,
          "stop" : 1728117753073,
          "duration" : 80
        }
      }, {
        "uid" : "e7474fdba93e03d1",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/e7474fdba93e03d1",
        "status" : "passed",
        "time" : {
          "start" : 1728109129756,
          "stop" : 1728109129826,
          "duration" : 70
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "3a7e7d0df25be538.json",
  "parameterValues" : [ ]
}